### PR TITLE
Added TimeRange and small fixes so master can compile

### PR DESCRIPTION
**Pull request by apluscs**
- Added empty TimeRange class
- Changed packages for Scheduledlnterview, Availability classes

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]
LGTM

**Comment by acmcarther** [comment]
Nits:
,
1. Please ensure the PR title is a complete statement, starting with a verb. In this case, your PR title is actually _misleading_ if read as if it complied with convention.

2. Keep the PR description updated with summarized details about the changes.

File: src/main/java/com/google/sps/data/DatastorePersonDao.java [hunk 1]
```diff
@@ -60,8 +60,8 @@ private static Entity personToEntity(Person person) {
   }
 
   /**
-   * Retrieve the person from Datastore from their email and wrap it in an 
-   * Optional. If they aren't in Datastore, the Optional is empty.
+   * Retrieve the person from Datastore from their email and wrap it in an Optional. If they aren't
```
Comment (acmcarther) [inline src/main/java/com/google/sps/data/DatastorePersonDao.java]: These changes are not relevant.

**Review by acmcarther — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/sps/data/DatastorePersonDao.java [hunk 2]
```diff
@@ -1,88 +0,0 @@
...
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.gson.Gson;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-
-/** Accesses Datastore to support managing Person entities. */
-public class DatastorePersonDao implements PersonDao {
```
Comment (acmcarther) [inline src/main/java/com/google/sps/data/DatastorePersonDao.java]: Why is this file being deleted?

Comment (apluscs) [reply src/main/java/com/google/sps/data/DatastorePersonDao.java]: l just removed it from tracking. l guess to Git it looks like it's deleted. My b.

**Review by acmcarther — APPROVED** [review]
LGTM

(I'll merge this to fix master)

**Pull request merged**